.github/workflows/build-deploy.yml

```diff
@@ -0,0 +1,81 @@
+name: Build and upload to PyPI
+
+# publish when a (published) GitHub Release is created:
+on:
+  push:
+  pull_request:
+  release:
+    types:
+      - published
+
+env:
+  # don't build wheels for unsupported CPython versions, and skip pypy altogether
+  CIBW_SKIP: cp27-* cp35-* pp*
```

Comment:
for future reference, see https://cibuildwheel.readthedocs.io/en/stable/options/#build-skip